contracts/CO30_Andrew_HotInHerre.ruha/edit_me/briefing/situation.html

```diff
@@ -0,0 +1,26 @@
+Ruha has seen inner turmoil. We've received a secondary contract to assist them in their time of need.
+
+<font color='#e3d310'>EMPLOYER</font>
+Ruha Government
+The Ruha Government is desperate for assistance. They're begging for any help as they have no one to assist their militia and law enforcement agency.
+
+<font color='#d81717'>ENEMY</font>
+Looters & Rioters
+Known to be anti-government civilians, they are defined as enemy. Armed and possibly dangerous, they will not give a second thought to kill anyone in their way for their gain. Dressed like a normal civilian, these anti-government individuals will be armed with basic firearms and use guerrilla tactics.
```

Comment:
```suggestion
Known to be anti-government civilians, they are defined as enemy. They will not give a second thought to kill anyone in their way for their gain. Dressed like normal civilians but will be armed with basic firearms and guerrilla tactics.
```
This file is too long, and a lot of this can be simplified to just provide the info the players need